typescript/cli/src/config/warp.ts

```diff
@@ -107,6 +107,13 @@ export async function createWarpRouteDeployConfig({
   outPath: string;
   advanced: boolean;
 }) {
+  // Providing the global --yes flag will trigger the usage of a trusted ISM while the
+  // --advanced flag will force the user to manually configure an ISM. Both flags should
+  // not be set to true at the same time.
+  if (context.skipConfirmation && advanced) {
+    throw new Error('Arguments advanced and yes are mutually exclusive');
+  }
```

Comment:
this feels like an annoying failure mode even though its more "explicit", can we just bias to one?

Reply:
Yep! I'll remove this check and if both flags have been provided then the `--advanced` flag will have priority